web/src/tippyjs.ts

```diff
@@ -417,6 +417,18 @@ export function initialize(): void {
         },
     });
 
+    delegate("body", {
+        target: "#user_message_content_in_email_notifications_label",
+        content: $t({
+            defaultMessage:
+                "Including message content in message notification emails is not allowed in this organization.",
+        }),
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
```

Comment:
The tooltip is shown even when the setting is not disabled. Please look at existing examples in the codebase where we show tooltip only for disabled cases.

Reply:
fixed in the latest commit